src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Because the return code determines whether to install the FreeDOS or non-FreeDOS set of config.sys+autoexec.bat as this is done from the code in install.c. I could also move that into the script?

I agree about C: VS D:, I don't know if there's any advantage for FreeDOS being in D: either.